src/views/Timeline.vue

```diff
@@ -119,11 +120,22 @@
                     align="top"
                     class="font-weight-regular pa-4 ma-6 see-thru"
                   >
+                    <div v-if="sample.args.inference_meta.display === 'Fall Detection'">
```

Comment:
@vickywane how about adding a few test cases to cover correct display of icons based on timeline event content?

Reply:
Adding a test case would mean that I have to connect to an edge device using a valid Peer ID, for the timeline to fetch fall events.

Reply:
Is there a way we can have a valid PeerID to use? 

That way I can boost the codecov report of this PR

Reply:
That is incorrect. We have discussed this topic before. When the focus is on testing a specific component logic, all moving parts not directly used in the tested logic can be isolated. The classic unit test vs integration test vs end to end test discussion. In this case we are looking at a vuetify component unit test.

In the pre-test section, pre-populate vuex with values for the various timeline event objects and then test that the icon component renders the correct icon values bases on the provided timeline event data object.

Reply:
> Is there a way we can have a valid PeerID to use?
> 
> That way I can boost the codecov report of this PR

Yes. Any UUID4 value would do. Just grab a fixed PeerID from somewhere and use it in tests.